src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -733,6 +733,38 @@ namespace winrt::TerminalApp::implementation
         {
             _noMatchesText().Visibility(Visibility::Collapsed);
         }
+
+        if (_currentMode == CommandPaletteMode::CommandlineMode)
+        {
+            ParsedCommandLineText(L"");
+
+            const auto commandLine = _getTrimmedInput();
+            if (!commandLine.empty())
+            {
+                ExecuteCommandlineArgs args{ commandLine };
+                ::TerminalApp::AppCommandlineArgs appArgs;
+                appArgs.DisableHelpInExitMessage();
+
+                if (appArgs.ParseArgs(args) == 0)
+                {
+                    const auto& commands = appArgs.GetStartupActions();
+                    if (commands.size() > 0)
+                    {
+                        std::wstring commandDescription{ RS_(L"CommandPalette_ParsedCommandLine") + L":" };
+                        for (const auto& command : commands)
+                        {
+                            commandDescription += L"\n\t" + command.Args().GenerateName();
+                        }
+                        ParsedCommandLineText(commandDescription.data());
+                    }
+                }
+                else
+                {
+                    ParsedCommandLineText(RS_(L"CommandPalette_FailedParsingCommandLine") + L":\n\t" + til::u8u16(appArgs.GetExitMessage()));
```

Comment:
Same down here, we should use `fmt::format` to add the error text to the resource here.

`NoticeFontNotFound` in the `TerminalControl` project is a good example

Reply:
Same as above